CHANGES.rst

```diff
@@ -4,6 +4,9 @@ History
 0.2.1 (unreleased)
 ------------------
 
+- Add latest commands in README.md
```

Comment:
Don't use wording like these, no body except you, knows what the latest changes are. Please write excactly what you did.